android/libraries/rib-base/src/main/java/com/badoo/ribs/core/ExperimentalApi.kt

```diff
@@ -0,0 +1,7 @@
+package com.badoo.ribs.core
+
+import javax.inject.Scope
+
+@Scope
+@Retention(AnnotationRetention.SOURCE)
+annotation class ExperimentalApi
```

Comment:
Why scope?
Maybe use https://kotlinlang.org/docs/reference/opt-in-requirements.html

Reply:
Will remove scope.

Let's consider kotlin opt-in after big review when we see what's actually experimental. Otherwise good idea.